util/util.go

```diff
@@ -0,0 +1,61 @@
+package util
```

Comment:
please try to avoid utility packages: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_avoid_package_names_like_base_common_or_util